### ISSUE TITLE
Meteorological data for 2021

### ISSUE THREAD

**Issue by daterma** [GIS4WRF/gis4wrf#228]
## GIS4WRF Issue
ds083.3 meteorological data in the Dataset> Met menu does not allow to download data for the year 2021 and part of 2020.

I manually downloaded the data NCEP GDAS 0.25 Degree Analysis Grids for 2021 on at the following link https://rda.ucar.edu/datasets/ds083.3/index.html#sfol-wl-/data/ds083.3?g=2021.

## WRF Issue
<details>
With this data I ran Geogrib, Ungrib, Metgrid and Real without problems, while Run WRF gave me the following error: 

Quilting with   1 groups of   0 I/O tasks.
 Ntasks in X            2 , ntasks in Y            2
*************************************
Configuring physics suite 'conus'

         mp_physics:      0*     0*
         cu_physics:      0*     0*
      ra_lw_physics:      4      4
      ra_sw_physics:      4      4
     bl_pbl_physics:      2      2
  sf_sfclay_physics:      2      2
 sf_surface_physics:      2      2

(* = option overrides suite setting)
*************************************
WRF V4.0 MODEL
 *************************************
 Parent domain
 ids,ide,jds,jde            1          31           1          31
 ims,ime,jms,jme           -4          22          -4          22
 ips,ipe,jps,jpe            1          15           1          15
 *************************************
DYNAMICS OPTION: Eulerian Mass Coordinate
   alloc_space_field: domain            1 ,              27804260  bytes allocated
 RESTART run: opening wrfrst_d01_2021-01-01_00_00_00 for reading
-------------- FATAL CALLED ---------------
program wrf: error opening   wrfrst_d01_2021-01-01_00_00_00 for reading
-------------------------------------------

System Information
 - OS: Windows 10
 - QGIS: 3.22.2

**Comment by daterma** [comment]
Any ideas to fix it?
Thanks

**Comment by dmey** [comment]
You are repoting two uncorrelated issues. I can have a look at the first next week --sorry no time this week. The second seems a WRF configuration issue, not something to do with GIS4WRF. I would recommend checking how to configure namelists in the [WRF users' guide](https://www2.mmm.ucar.edu/wrf/users/docs/user_guide_v4/contents.html). If you are still having issues after that, try to ask someone on the [WRF & MPAS-A Support Forum](https://forum.mmm.ucar.edu/phpBB3/).

**Comment by daterma** [comment]
OK, thanks a lot.
I await your information on the first problem.

**Comment by letmaik** [comment]
Correction: The API we use currently seems to have been replaced by a new one in RDA that is JSON-based. The old API is still functional but is not updated with new data anymore. So the solution would be to transition gis4wrf to the new API: https://rda.ucar.edu/#!apps_api_desc

**Comment by daterma** [comment]
Thanks for the support.
Do we have to wait for GIS4WRF update?

**Comment by letmaik** [comment]
No, go to Simulation -> Data tab and import your own data with "Use Custom Dataset" at the bottom.

**Comment by letmaik** [comment]
@daterma The new release is out now and uses the new RDA API. If you find a bug, please open a new issue. Thanks again for reporting!
